include/ck_tile/core/tensor/static_distributed_tensor.hpp

```diff
@@ -192,6 +202,29 @@ set_tile_if(static_distributed_tensor<DataType, StaticTileDistribution>& out_ten
     });
 }
 
+template <typename DataType, typename StaticTileDistribution, typename XIndicesPredicate>
+CK_TILE_HOST_DEVICE void set_tile_if(
+    static_distributed_tensor<DataType, StaticTileDistribution>& out_tensor,
+    DataType value,
+    XIndicesPredicate predicate,
+    decltype(detail::get_partition_index(std::declval<StaticTileDistribution>())) partition_index)
```

Comment:
Since `partition_index` is used by this public API (`set_tile_if`),   suggest we also add `get_partition_index` as a public API

Reply:
Moved the `get_partition_index()` to the `ck_tile` namespace